### ISSUE TITLE
No systray for agent-gui

### ISSUE THREAD

**Issue by Nimamoh** [rupor-github/win-gpg-agent#10]
Hello,

When launching agent-gui.exe, I have no systray icon nor do I have any error poping. I tried launching it with '-d' option but no logs whatsoever.

From process manager, I see the 'GUI wrapper for gpg-agent' for a few second before it disappearing

EDIT: using release 1.3.2 from release zip

**Comment by rupor-github** [comment]
There are many possible reasons this could happen, simplest one being you already have instance hung is a limbo state. Take a look in a Task Manager or using command line do something like this:
```
> tasklist | findstr agent
agent-gui.exe                 2416 Console                    1     15,864 K
gpg-agent.exe                13972 Console                    1      8,792 K
```
And kill everything before re-trying.

Now - all debug output is being reported using OutputDebugString(), so you need something like [Sysinternals DebugView](https://docs.microsoft.com/en-us/sysinternals/downloads/debugview) to see what is going on.

**Comment by rupor-github** [comment]
Now you have to figure out where your installed gnupg creates those, read project README and create proper configuration file for your installation.

**Comment by rupor-github** [comment]
This is now fixed in v1.4.0. Please, see updated README for details as it may be breaking change if you had non-default configuration file.
